packages/preview/fletcher/0.5.1/docs/gallery/algebra-cube.typ

```diff
@@ -0,0 +1,47 @@
+#import "@preview/fletcher:0.5.0" as fletcher: diagram, node, edge
```

Comment:
Examples in the gallery still import the previous version, maybe you want to update it here as well.

Reply:
Ah, thanks. Every time I forget this and you remind me! Will fix...

Reply:
I like to use [tbump](https://github.com/your-tools/tbump) for exactly this reason. You configure it once and then upgrade all references to the current version at once.

Reply:
All done. And thanks @jneug, I'll definitely use that